JournalCoreData/Model Controllers/EntryController.swift

```diff
@@ -107,7 +108,7 @@ class EntryController {
                 completion(NSError())
                 return
             }
-
+            
             let moc = CoreDataStack.shared.mainContext
```

Comment:
This needs to be on a background context, not the main one. `let moc = CoreDataStack.shared.container.newBackgroundContext()`